Rogue-Ape-Studios-XR/Secrets-of-Ignacios
# ✨player rig changes

## Content

Added functionality for new VFX, Inventory and grimoire button to main player rig

## Changes
### Updated
`Assets/Packs/Inventory System/Scripts/Inventory.cs` : Was updated. No longer applies/removes kinematic.
`Assets/Prefabs/PlayerRig/PlayerRig.prefab` : Was updated. Added references
`Assets/Scripts/Gestures/ScriptableObjects/Air.asset` : Was updated. With water vfx

### Removed
`Assets/Prefabs/PlayerRig/PlayerRig with fixed book.prefab` : Was removed. No longer needed

## Testing

The feature / Bugfix can be testing by following these steps:

1. Open main scene
2. Look at wrist
3. Try casting and see VFX stuff
4. Try inventory that is on right wrist
